exercises/D-logical-operators/exercise.js

```diff
@@ -5,20 +5,21 @@
   The variables should have values that match the expected results.
 */
 
+
 // Do not change these two statement
 let htmlLevel = 8;
 let cssLevel = 4;
 
 // Finish the statement to check whether HTML, CSS knowledge are above 5
 // (hint: use the comparison operator from before)
-let htmlLevelAbove5;
-let cssLevelAbove5;
+let htmlLevelAbove5 = true;
+let cssLevelAbove5 = false;
```

Comment:
And the same is true for the two previous statements. What should the expression be?